README.md

```diff
@@ -6,6 +6,22 @@ FIXME: description
 
 Download from http://example.com/FIXME.
 
+### With Docker
```

Comment:
:)

Reply:
We have to right something in the README eventually :)

Reply:
definitely !!
that is on my todo list as well